README.md

```diff
@@ -1,8 +1,45 @@
+vpaw by Kitware, Inc.
+================================
+A Virtual Pediatric Airways Workbench (VPAW) is a patient-centered surgical planning software system targeted to pediatric patients with airway obstruction. VPAW enables a  full surgical planning pipeline, which includes: importing DICOM images, segmenting the airway, interactive 3D editing of airway geometries to express potential surgical treatment planning options, and creating input files for offline geometric analysis and computationalfluid dynamics simulations for evaluation of surgical outcomes.  This repository contains a new generation of VPAW that is developed using [3D Slicer](https://www.slicer.org/).
 
-A Virtual Pediatric Airways Workbench (VPAW) is a patient-centered surgical planning software system targeted to pediatric patients with airway obstruction. VPAW is enables a  full surgical planning pipeline, including importing DICOM images, segmenting the airway, interactive 3D editing of airway geometries to express potential surgical treatment planning options, and creating input files for offline geometric analysis and computationalfluid dynamics simulations for evaluation of surgical outcomes.  This repostiory contains a new generation of VPAW that is developed using [3D Slicer](https://www.slicer.org/).
+_This project is in active development and may change from version to version without notice,_
 
+## Features
 
+a) Importing DICOM images \
+b) Segmenting the airway \
+c) Interactive 3D editing of airway geometries to express potiential surgical treatment planning options \
+d) Creating input files for offline geometric analysis and CFD simulations
```

Comment:
```suggestion

The project is being actively developer and our plan is to implement the following functionalities to enable users to express potential surgical treatment planning options for pathology like XX, YYY.

A typical workflow will consist in the following steps:

* Import of DICOM images
* Segmentation of airway
* Interactive 3D editing of airway geometries leveraging integration of XXX device through OpenHaptics interface.
* Export of segmented airways for offline geometric analysis and CFD simulations
```

Reply:
@jadh4v Review the description for workflow for accuracy and update accordingly :pray:

Reply:
> Import of DICOM images

Is there a specific set of modalities that we plan to support first (CT, MRI, ... ?)

Reply:
> Segmentation of airway

Which approach will be available ? Full set of effect available in Slicer segmentation editor ? Or a specific subset ? Or a novel method specific to this project ?

Reply:
> > Import of DICOM images
> 
> Is there a specific set of modalities that we plan to support first (CT, MRI, ... ?)

The reference paper itself only refers to CT scans. MRI is usually more expensive and time consuming as compared to CT. I suspect that CT scans are probably preferred for this treatment. In the ReadMe, we can say:
`* Import of DICOM images (initially we will support only CT data)`

Regarding the segmentation methods included, I think this is something of a future decision and will require input from collaborators on whether we are going to modify the previously established/published approach. For now we could go with what the published paper says. It applies the method described by _Hong, et al._ in _section 2.1_ of https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4769591/

Reply:
@andinet do you want to pitch in on these points?